dcos_launch/config.py

```diff
@@ -542,3 +554,19 @@ def deduce_image_project(doc: dict):
         'type': 'boolean',
         'required': False,
         'default': False}}
+
+
+TERRAFORM_SCHEMA = {
+    'dcos-enterprise': {
+        'type': 'boolean',
+        'default': False},
+    'platform': {
```

Comment:
Because you need to implement the sub-versions of terraform then it would be nice to validate that the platform is one of the allows (i.e. implemented) values

Reply:
Done